### PR TITLE
add mic pod exception to helm chart

### PR DESCRIPTION
**Pull request by aramase**
**Reason for Change**:

When NMI runs on the same node as MIC pod, then MIC request to fetch token is intercepted by NMI. This token request will fail pod-identity validation as MIC is yet to initialize and NMI can't find an `AzureAssignedIdentity` for MIC. With this change, we install the `AzurePodIdentityException` for MIC pod labels so token requests from MIC go directly to IMDS instead of pod-identity validation path.

**Issue Fixed**:

fixes https://github.com/Azure/aad-pod-identity/issues/484

**Notes for Reviewers**:

### CODE REVIEW

File: charts/aad-pod-identity/values.yaml [hunk 1]
```diff
@@ -14,6 +14,13 @@ image:
 # To match only pods in the namespace containing AzureIdentity set this to true.
 forceNameSpaced: "false"
 
+# When NMI runs on a node where MIC is running, then MIC token request call is also
+# intercepted by NMI. MIC can't get a valid token as to initialize and then
+# assign the identity. Installing an exception for MIC would ensure all token requests
+# for MIC pods directly go to IMDS and not go through the pod-identity validation
+# https://github.com/Azure/aad-pod-identity/blob/master/docs/readmes/README.app-exception.md
+installMICException: "true"
```
Comment (kkmsft) [inline charts/aad-pod-identity/values.yaml:22]: Is this required by default ? Wouldn't this only be for system assigned identity clusters ?

File: charts/aad-pod-identity/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Deploy components for aad-pod-identity
 name: aad-pod-identity
-version: 1.5.5
+version: 1.5.6
```
Comment (kkmsft) [inline charts/aad-pod-identity/Chart.yaml:4]: Is this going to be updated right now or near the release ?

File: charts/aad-pod-identity/values.yaml (hunk 1 above)
Comment (aramase) [reply charts/aad-pod-identity/values.yaml:22]: This is required for all helm installs for AKS clusters. In AKS clusters, NMI runs on the same node as MIC and fails to fetch tokens.

File: charts/aad-pod-identity/Chart.yaml (hunk 2 above)
Comment (aramase) [reply charts/aad-pod-identity/Chart.yaml:4]: This is just chart version. I can make a tar file in this PR or create a new one with that? wdyt?

**Pull request merged**